features/vendor_views_closed_auction.feature

```diff
@@ -38,6 +38,11 @@ Feature: Vendor views closed auctions
     And I should see when the auction ended
     And there should be meta tags for the closed auction
 
+  Scenario: Winning vendor was paid
```

Comment:
Yeah this feature is for anyone, guest or vendor -- will put in a different file to make that more clear